### ISSUE TITLE
[330007] delegates of objc Textviews do not react to spaces from external keyboards(possibly related to #587)

### ISSUE THREAD

**Issue by bennr01** [omz/Pythonista-Issues#597]
**TL;DR:** Delegates of objc do no not get their handlers (like `textview_did_change()`) called when the change/event is a space (`" "`) entered by an external keyboard.

**Long description:** Delegates of Objc-instances of `SUITextView` recently stopped getting their delegate methods (e.g. `textview_should_change()`, `textview_did_change()`, ...) called for the space-key on external keyboards. This bug does not exists on the on-screen keyboard and does also not affect any other keypress. This bug may be older than 330007, but is definitely new with the beta. It may be related to #587, but that bug report focuses on the general pythonista UI.
This bug was initialy reported on the pythonista forum as a bug in StaSh, but since the bug only exists in the beta and the behaviour just seems too inconsistent (why should the internal keyboard work just fine?), it seems to be more appropiate to post it here. One of the forum threads is [here](https://forum.omz-software.com/topic/5381/stash-alternatives-for-pip) (only a short mention that the space-key stopped working on external keyboards). The bug was also mentioned on Slack and confirmed by other users.

**Steps to reproduce:** The easiest way to reproduce would simply be launching StaSh and using an external keyboard.

**Sample code:** I am not experienced with `objc_util`, so i am unable provide some minimal code, but you can find an example delegate [here](https://github.com/ywangd/stash/blob/6506b0b50ada0e385c58cba0dc0d6c1aee735742/system/shuseractionproxy.py#L42) and initialization and installation of the delegate [here](https://github.com/ywangd/stash/blob/6506b0b50ada0e385c58cba0dc0d6c1aee735742/system/shterminal.py#L220) and [here](https://github.com/ywangd/stash/blob/6506b0b50ada0e385c58cba0dc0d6c1aee735742/system/shterminal.py#L269). Adding `print`-statements to the delegate reveals that they are not called for any  whitespaces on the external keyboard, but getting called for non-space-keys and spaces on the on-screen keyboard.

**System Information:**
Pythonista 3.3 (330007), Default Interpreter 2.7.12
iOS 12.1.3, model iPhone8,1, resolution (portrait) 750.0 x 1334.0 @ 2.0

**Edit:** I seem to be unable to add the correct lables to this Issue. It would be a normal bug, but i am unsure whether this is related to an internal-module or pythonista itself (given the recent changes to the keyboard).

**Comment by jsbain** [comment]
i wonder if the objc version of the delegates have the same issue?

**Comment by bennr01** [comment]
This issue seems to be fixed with 330012 :)
